src/api/preferences/put-preference.api.ts

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios'
+import { CustomizedAxiosResponse } from '../index.interface'
+import { IPreference } from './index.interface'
+
+export const putPreferenceApi = async (): Promise<
+  CustomizedAxiosResponse<IPreference>
```

Comment:
put requires `params` (the data you want to modify to)
https://github.com/ajktown/wordnote/blob/main/src/api/words/put-word-by-id.api.ts#L7